test/unit/services/test-contracted-hours-service.js

```diff
@@ -0,0 +1,85 @@
+const expect = require('chai').expect
+const assert = require('chai').assert
+const sinon = require('sinon')
+require('sinon-bluebird')
+const proxyquire = require('proxyquire')
+const orgUnitConstant = require('../../../app/constants/organisation-unit.js')
+const orgUnitFinder = require('../../../app/services/helpers/org-unit-finder')
+const breadcrumbHelper = require('../../helpers/breadcrumb-helper')
+
+const CONTRACTED_HOURS = 37.5
+const UPDATED_CONTRACTED_HOURS = 22
+
+var id = 1
+var breadcrumbs = breadcrumbHelper.OFFENDER_MANAGER_BREADCRUMBS
+var expectedTitle = breadcrumbs[0].title
+var expectedSubTitile = orgUnitFinder('name', orgUnitConstant.OFFENDER_MANAGER.name).displayText
+
+var contractedHoursService
+var getBreadcrumbs
+var getContractedHoursForWorkloadOwner
+var updateContractedHoursForWorkloadOwner
+
+beforeEach(function () {
+  getContractedHoursForWorkloadOwner = sinon.stub()
+  updateContractedHoursForWorkloadOwner = sinon.stub()
+  getBreadcrumbs = sinon.stub().returns(breadcrumbs)
+  contractedHoursService =
+    proxyquire('../../../app/services/contracted-hours-service',
+      {
+        './data/get-contracted-hours-for-workload-owner': getContractedHoursForWorkloadOwner,
+        './data/update-contracted-hours-for-workload-owner': updateContractedHoursForWorkloadOwner,
+        './get-breadcrumbs': getBreadcrumbs
+      })
+})
+
+describe('services/contracted-hours-service', function () {
```

Comment:
Worth adding a nested `describe` here to call out which function within the service the tests are for

Reply:
done